### PR TITLE
pkp/pkp-lib#3302 clearCache needed after these changes 

### PR DESCRIPTION
**Pull request by bozana**
s. https://github.com/pkp/pkp-lib/issues/3302

Is this way to clear cache enough? -- when the order of the grid columns changed...

### CODE REVIEW

File: classes/install/Upgrade.inc.php [hunk 1]
```diff
@@ -40,6 +40,18 @@ function isUpgrade() {
 	// Specific upgrade actions
 	//
 
+	/**
+	 * Clear the data cache files (needed because of direct tinkering
+	 * with settings tables)
+	 * @return boolean
+	 */
+	function clearDataCache() {
```
Comment (asmecher) [inline classes/install/Upgrade.inc.php]: This function is already in the parent class.

**Comment by asmecher** [comment]
The order of the grid columns changing isn't cached -- they're constructed in PHP, and the ordering isn't reflected in any of the templates.

**Comment by bozana** [comment]
hmm... do you have an idea why I then had to clear cache in order to have the correct data in the supportedSubmissionLocales array -- before clearing the cache there was always a double locale there... Hmmm... :-)

**Comment by asmecher** [comment]
> before clearing the cache there was always a double locale there...

Hmm, I can't think of any reason for this -- but a cache clear won't have negative side-effects, so I'm content to leave it as it is.

**Pull request merged**